app/src/main/res/values/strings.xml

```diff
@@ -283,4 +283,5 @@
     <string name="notification_channel_name">"General"</string>
     <string name="default_notification_channel_id">permanent_general_notification_channel</string>
     <string name="notification_body_share_notification">%s has shared %s with you</string>
+    <string name="version_text">Version %s (%s)</string>
```

Comment:
Maybe it would be better to have here instead of "Version" the app name + env so we can tell which environment we use